### PR TITLE
feat(create-wundergraph-app): download latest tag instead of main

### PR DESCRIPTION
**Pull request by fiam**
Downloading from the main branch prevents us from making breaking
changes, since the examples would stop working until the next SDK with
the breaking changes is released.

When using the wundegraph repository as the example source, first
retrieve the tags and use the latest tag starting with @wundergraph/sdk

### CODE REVIEW

File: packages/create-wundergraph-app/src/helpers/getRepository.ts [hunk 1]
```diff
@@ -22,16 +30,19 @@ const resolveRepository = async ({ exampleName, githubLink }: { exampleName?: st
...
-			const selectedExampleName = await checkIfValidExample(exampleName);
+			if (!ref) {
+				ref = await resolveLatestWundergraphRef();
+			}
+			const selectedExampleName = await checkIfValidExample(exampleName, ref);
 			return {
 				repoOwnerName: 'wundergraph',
 				repoName: 'wundergraph',
-				branch: 'main',
+				ref: 'main',
```
Comment (JivusAyrus) [inline packages/create-wundergraph-app/src/helpers/getRepository.ts:45]: Shouldn't this be ' ref: ref || "main" '

**Review by JivusAyrus — CHANGES_REQUESTED** [review]

**Review by Pagebakers — APPROVED** [review]
Nice one!

File: packages/create-wundergraph-app/src/helpers/getRepository.ts (hunk 1 above)
Comment (fiam) [reply packages/create-wundergraph-app/src/helpers/getRepository.ts:45]: You're right, good catch. I'll update it.

Comment (fiam) [reply packages/create-wundergraph-app/src/helpers/getRepository.ts]: Fixed, PTAL

**Review by JivusAyrus — APPROVED** [review]
LGTM

**Pull request merged**